app/commands/v2/discard_draft.rb

```diff
@@ -10,14 +10,13 @@ def call
 
         delete_supporting_objects
         delete_draft_from_database
-        delete_draft_from_draft_content_store(draft_path) if downstream
+        delete_draft_from_draft_content_store(draft_path)
```

Comment:
Yes cool, I had it in there originally, don't know why I moved it out again. 
It is back now